.github/workflows/cdk_deploy.yaml

```
@@ -119,6 +123,12 @@ on:
         required: false
         type: string
         default: Dockerfile
+      S3_BUCKET_NAME_PREFIX:
+        required: false
+        type: string
+      S3_TTL_IN_DAYS:
```

```suggestion
      S3_TTL_SPEC:
```

Also suggest documenting the format here, as this is kind of an interface declaration for the deployment step.